src/index.js

```diff
@@ -25,7 +25,6 @@ function extend(base, props) {
 
 function setClass(attributes, value, append) {
 	let cl = getClass(attributes);
-	if (attributes.className) delete attributes.className;
```

Comment:
just wondering if it overwrites preact-mdl's class modifications.  I can set up a test if needed